openmdao/components/indep_var_comp.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, name, val=None, **kwargs):
                              "`str` or an iterable of tuples of the form (name, value) or "
                              "(name, value, keyword_dict).")
 
-    def apply_linear(self, mode, ls_inputs=None, vois=(None, ), gs_outputs=None):
+    def sys_apply_linear(self, mode, ls_inputs=None, vois=(None, ), gs_outputs=None):
```

Comment:
That's not really how we've been using underscores so far.  We tend to use them on functions that we don't want to show up in the generated docs, i.e., functions that are not user facing, so in this case, using an underscore would be consistent with what we've been doing.